MaintenanceDashboard.Data/API/CheckValue.cs

```diff
@@ -6,19 +6,14 @@ static class CheckValue
     {
         public static void RequireString(string value)
         {
-            //TODO: Add any MassageBox
-            if (value == null)
-                throw new ArgumentNullException();
-            else if (value.Trim().Length == 0)
-                throw new ArgumentException();
+            if (value == null || value.Trim().Length == 0)
+                throw new ArgumentNullException(nameof(value));
         }
 
         public static void RequireForeignKey(int value)
         {
-            if (value == null)
```

Comment:
int nigdy nie będzie nullem. 

Jeśli chcesz, by int byłl `nulable` tworzysz coś takiego:

```csharp
public int? JakasZmienna {get;set;}
```

znak zapytania mówi kompilatorowi, że ten typ wartościowy (takie jak char, bool, double itd) mogą przyjąć wartośc `null`. Jesli nie ma zapisu `int?` int nigdy nie będzie nullem. Jeśli nie przypiszesz wartości do `int` to `value` będzie miało wartość `0` bo taki jest jej defaultowa wartość. `double` też będzie zerem. Domyślnie `bool` jest false